### DISCUSSION TITLE
esp32Wroom32 in Betriebnahme

### DISCUSSION THREAD

**Discussion by MG192529** [tbnobody/OpenDTU#2142] [Q&A]
Guten Tag,
ich bin vollkommener Laie. Nun möchte ich einen esp32Wroom32 in betrieb nehmen (zur Abfrage HM600) ich habe mich bei You Tube umgesehen
https://www.youtube.com/watch?v=PpjCz33pGkk find aber die dort genannten BIN Dateien nicht.
Ich bitte um Hilfe
esp32Wroom ,NRF24L01+NRFL01,ext.Antenne
Danke

**Comment by Juergen2453** [comment]
zum Beispiel:
https://github.com/tbnobody/OpenDTU/actions/runs/9725623648

**Comment by oe3osb** [comment]
Hier gibt's auch noch gute Erklärungen:
[https://www.opendtu.solar/firmware/](url)

**Comment by MG192529** [comment]
Vielen Dank. Wenn Ich wieder in De bin werde ich loslegen

14.07.2024 13:21:55 Erich Varga ***@***.***>:
